addons/udes_sale_stock/models/sale_order.py

```diff
@@ -299,11 +299,11 @@ def action_confirm(self):
         """ Override to disable tracking by default """
         return super(SaleOrder, self.with_context(tracking_disable=True)).action_confirm()
 
-    def confirm_orders(self):
+    def confirm_orders(self, states=["draft"]):
```

Comment:
Not sure if this is likely to cause an issue here but it can be dangerous using mutable objects as default arguments.

Might be safer to have states default as `None` and then at the start of the method do:

```
if states is None:
    states = ["draft"]
```

Reply:
No longer needed after the change to remove the optional argument